# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: devtools/client/webconsole/test/browser/_webconsole.toml
# Commit: ff5ea82f6090
# Full Hash: ff5ea82f6090a4a1645a1a32749a4ae9102f015b
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-04-17 16:30:04
# Regressor Bug: 1957355
# File Overlap Count: 1
# Description:
#   Bug 1957355 - No storage access for resources injected by content-scripts, r=timhuang,necko-reviewers,devtools-reviewers,anti-tracking-reviewers,nchevobbe,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D244204
# ==============================================================================

diff -r 04eed8759454 -r ff5ea82f6090 devtools/client/webconsole/test/browser/_webconsole.toml
--- a/devtools/client/webconsole/test/browser/_webconsole.toml	Thu Apr 17 10:56:59 2025 +0000
+++ b/devtools/client/webconsole/test/browser/_webconsole.toml	Thu Apr 17 10:56:59 2025 +0000
@@ -7,6 +7,8 @@
   "code_bundle_nosource.js",
   "code_bundle_nosource.js.map",
   "cookieSetter.html",
+  "empty-with-cookie.js",
+  "empty-with-cookie.js^headers^",
   "head.js",
   "shared-head.js",
   "sjs_cors-test-server.sjs",
@@ -20,6 +22,8 @@
   "test_hsts-invalid-headers.sjs",
   "test-batching.html",
   "test-blank.html",
+  "test-blank-with-cookie.html",
+  "test-blank-with-cookie.html^headers^",
   "test-console-trace-duplicates.html",
   "test-console-api-iframe.html",
   "test-console-api.html",